conf/container.config

```diff
@@ -0,0 +1,20 @@
+process {
+    withLabel: simulation_image {
+        container = "olivierlabayle/targene-simulations:0.0"
+    }
+    withLabel: targenecore_image {
```

Comment:
Should this be set to a new version release of tl-core rather than a branch?

Reply:
Indeed, I was waiting for a potential review on the TargeneCore repo but merged it now and updating these tags.